1-org/envs/shared/projects.tf

```diff
@@ -410,3 +413,23 @@ resource "google_project_iam_member" "network_sa_restricted" {
   role    = each.key
   member  = "serviceAccount:${local.networks_step_terraform_service_account_email}"
 }
+
+/******************************************
+  Import Cloud Build and Seed Projects
+*****************************************/
+
+resource "google_access_context_manager_service_perimeter_resource" "cloudbuild_project" {
+  count          = local.enforce_vpcsc ? 1 : 0
+  perimeter_name = "accessPolicies/${var.access_context_manager_policy_id}/servicePerimeters/${local.perimeter_name}"
+  resource       = "projects/${local.cloudbuild_project_number}"
+
+  depends_on = [module.service_control]
+}
+
+resource "google_access_context_manager_service_perimeter_resource" "seed_project" {
+  count          = local.enforce_vpcsc ? 1 : 0
```

Comment:
does this count mean that if the perimeter is in dry run the projects will not be added @mariammartins ?